tfgrid3/wordpress/zinit/ssh_config.yaml

```diff
@@ -0,0 +1,11 @@
+exec: /bin/sh -c "
+  mkdir -p /var/run/sshd;
+  mkdir -p /root/.ssh/;
+  chmod 600 ~/.ssh;
+  chmod 600 /etc/ssh/*;
+  touch ~/.ssh/authorized_keys;
+  chmod 600 ~/.ssh/authorized_keys;
+  echo $SSH_KEY > /root/.ssh/authorized_keys;
+  echo 'ssh configred'"
+test: stat /root/.ssh/authorized_keys
```

Comment:
Zinit oneshot services does not use the tests, if the service exited succesfully then it considered completed and it is okay to start any dependant services.
Tests are only started for long living zinit services.
see [here](https://github.com/threefoldtech/zinit/blob/82e13451be6c22102e430461a4f0fa34f5e6e265/src/zinit/mod.rs#L645)
So i recommended:
1. remove the redundant test as it will never executed in this case. also zinit not complain atm, but this could be change in the feature.
2. I understand that you want to ensure that the code executed successfully before starting the sshd, for that you can make sure that bash will exit with error on case if any if the command failed, because normally the script will always exit with the last command exit code, in your case `echo` will always succeded (unless you wrote to stream that suddenly fails).
To solve this, you can use `-e` on the command line.
You may also want to employ all or some of the the `-u` ,  `-o pipefail` (pipefail for `bash` only not for `sh`) options like so:
```
/bin/sh -eux -c " YOUR CODE "
```

notes:
- `-e` exits on error
- `-u` errors on undefined variables
- `-o` (for option) `pipefail` exits on command pipe failures [won't work with `sh`].

Reply:
resolved in [5b089d0](https://github.com/threefoldtech/tf-images/pull/125/commits/5b089d06d5d30ef07ff998da38dd6064b9f285e3)